src/Uno.UI.RemoteControl.VS/Uno.UI.RemoteControl.VS.csproj

```diff
@@ -1,6 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+	<!-- TODO: Figure out if we really need to multi-target this or if net48 is enough -->
```

Comment:
net461 is for vs2019.

Reply:
@jeromelaban But do we still care about VS 2019?

Reply:
For now yes, I should have mentioned. Let's keep it until it causes us build trouble.